src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/Airship.cs

```diff
@@ -43,12 +42,46 @@ private void Init()
                     handler(this, EventArgs.Empty);
                 }
             });
+
+            NSNotificationCenter.DefaultCenter.AddObserver(aName: (NSString)"com.urbanairship.airship_ready", (notification) =>
+            {
+                UAirship.Shared().WeakDeepLinkDelegate = this;
+            });
+
```

Comment:
Do you need to remove this line, now that you are handling it below?

Reply:
Thanks for noticing that. Fix on the way.